File: C Language/pro133.c
/**
    count how many digits in entered number
*/

#include<stdio.h>

void main(){
    int ip, ans = 0;

    printf("\n Enter any number \t");
    scanf("%d", &ip);

    while(ip > 0){
        ans++;
        ip = ip / 10;
    }

    printf("\n total digits are %d", ans);
}

File: PHP/pro133.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <?php
        /*
            The while loop does not run a specific number of times, but checks after each iteration if the condition is still true.

            The condition does not have to be a counter, it could be the status of an operation or any condition that evaluates to either true or false.

            find total digit in enetred number
        */ 

        $ip1 = 124;
        $ans = 0;

        while($ip1 >  0){
            $ans++;
            //$ip1 = round($ip1 / 10);
            $ip1 = floor($ip1 / 10);
            //$ip1 = (int)$ip1 / 10;
        }

        echo "<hr> Total Digits are $ans";
    ?>
</body>
</html>